examples/tv-app/tv-common/include/AppTv.h

```diff
@@ -150,7 +150,8 @@ class DLL_EXPORT ContentAppFactoryImpl : public ContentAppFactory
     // Remove the app from the list of mContentApps
     bool UninstallContentApp(uint16_t vendorId, uint16_t productId);
     // Print mContentApps and endpoints
-    void PrintInstalledApps();
+    void LogInstalledApps();
+    // TODO: method to retireve list of mContentApps
```

Comment:
Typo of `retrieve` and please add an issue number (any bare TODOs are discouraged).

Reply:
Created an issue: https://github.com/project-chip/connectedhomeip/issues/34020